### DISCUSSION TITLE
Anyone can share 0.6mm nozzle profile?

### DISCUSSION THREAD

**Discussion by Trilis29** [bassamanator/Sovol-SV06-firmware#139] [General]
I am looking for PS or Orca profile for 0.6mm nozzle, maybe someone could share? :)

**Comment by bassamanator** [comment]
The bundle on this repo should be parametric, so adjusting nozzle size to 0.6 should be all that is required.

**Reply by Trilis29** [reply]
extrusion width parameters are non percent based, so these won't change by changing the nozzle only or I miss something?

**Comment by bassamanator** [comment]
You're right, the PS settings are not percent based, but the SS settings are.

Shouldn't be too hard. Duplicate the Print settings profile you want to use, then convert the mm based values to percentages keeping in mind that these values are for a 0.4mm nozzle.
![image](https://github.com/bassamanator/Sovol-SV06-firmware/assets/61985779/eb82914d-02cb-46b8-b3e3-2e8a7a9eb2e3)

**Comment by JoernSchoenyan** [comment]
@Trilis29 what is your solution? If you didn't tinker around with it, I would volunteer to port the settings over to OrcaSlicer.

**Comment by JoernSchoenyan** [comment]
@bassamanator right now, there is a bug in Orcaslicer which prevents exporting the printer profile - when this is fixed, would you accept a pull request with a Orca profile?

**Reply by bassamanator** [reply]
Appreciate the offer! I must ask though, did you make your settings profile from scratch, or are your settings based on Ellis's? As might be apparent from this repo, and my posts online, I'm an Ellis kinda guy.

**Reply by JoernSchoenyan** [reply]
I took the Orcaslicer Marlin SV06 profile as a base, converted it to Klipper and transfered as much settings from your SuperSlicer profile as possible. I have a different naming scheme, though. I've done some prints already with good results, but I'm not yet at the speeds I've tried with your PrusaSlicer settings. As I progress testing, I'm increasing the speed step by step.

**Reply by bassamanator** [reply]
Thanks man, but I'm strictly an Ellis guy. I want users to be able to switch between the SS/PS/?? bundles, and know exactly what the print settings do based on their names.

#### Why am I strictly an Ellis guy?
His print settings took ~2 years to develop. All changes that he made are documented, meaning, there is very little guess work, if any.

#### For any pull requests relating to slicer bundles
- the naming convention must match Ellis's.
- All settings (as much as is possible) must match the config bundles (SS/PS; Ellis's print settings) found on this repo.

If you can adhere to this, feel free to create a PR. Regardless, I appreciate all contributions as well as _attempts_ at contribution. 🙏🏼

**Reply by JoernSchoenyan** [reply]
Very reasonable. When https://github.com/SoftFever/OrcaSlicer/issues/3803 is fixed, I'll look into it. Naming conventions shouldn't be much of an issue, testing very high speed settings could be a problem.

**Reply by bassamanator** [reply]
Well, if you're trying to port over his settings, then you should simply port them over, that applies to speed settings as well; no testing especially for speed is required. 

What do you have _your_ speeds set to? Just curious. I print with the 15mm3_coreXY-speeds profile as is, _except_ for max volumetric speed set to 17mm3/s.

**Reply by bassamanator** [reply]
Also feel free to tag me in for speed testing.

**Reply by JoernSchoenyan** [reply]
My current speed values: 
![image](https://github.com/bassamanator/Sovol-SV06-firmware/assets/1822665/34c0ff64-9413-4418-92dc-7ce39106c616)

But now, my bed thermistor is faulty, I think the cable is broken as it goes wild when moving. Need to investigate further.

**Reply by dweenz** [reply]
you can simply run 2 new wires to test. There have been a few people with this issue.

**Reply by JoernSchoenyan** [reply]
New wires did the trick.

**Reply by Trilis29** [reply]
@JoernSchoenyan could you upload your config?

**Reply by JoernSchoenyan** [reply]
The bug in OrcaSlicer isn't fixed yet, so I can't export the profile.
